app/src/main/kotlin/com/twain/interprep/presentation/ui/components/DropdownMenuInput.kt

```diff
@@ -0,0 +1,59 @@
+package com.twain.interprep.presentation.ui.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.*
+import androidx.compose.ui.Modifier
+import androidx.compose.runtime.*
+import androidx.compose.ui.tooling.preview.Preview
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun DropdownMenuInput(
+    modifier: Modifier,
+    options: List<String>,
+    labelText: String,
+) {
+    var expanded by remember { mutableStateOf(false) }
+    var selectedOptionText by remember { mutableStateOf("") }
+
+    ExposedDropdownMenuBox(
```

Comment:
In the textinput fields where dropdown is shown , user should also be allowed to type. Are we allowing now?

Reply:
I'll make this change in a separate PR to unblock